### PR TITLE
Add organization_id to group table

### PR DESCRIPTION
**Pull request by hmstepanek**
Add a db migration script to add organization_id to group table.

### CODE REVIEW

**Comment by hmstepanek** [comment]
I don't think I need to add the organization as that is a relationship rather than a column in the db. Lmk if I'm thinking about that wrong.

**Comment by robertknight** [comment]
> I don't think I need to add the organization as that is a relationship rather than a column in the db. Lmk if I'm thinking about that wrong.

You're right about this. The `organization` attribute is purely part of SQLAlchemy's model and not actually present in the DB schema, although the `organization_id` column is used to resolve it.

File: h/migrations/versions/41b1f338a_add_organization_to_group_model.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+revision = '41b1f338a'
+down_revision = 'af88524994ce'
+
+
+def upgrade():
+    op.add_column('group', sa.Column(
+        'organization_id',
+        sa.Integer,
+        sa.ForeignKey('organization.id', ondelete='cascade'),
```
[the remark this answers is not in the document]
Comment (seanh) [reply h/migrations/versions/41b1f338a_add_organization_to_group_model.py]: At some point we'll make `group.organization_id` not nullable, so it won't be possible to delete an organization and leave behind groups with no organization. You would have to delete all the groups (and whatever that involves) first and then delete the organization. Or move the groups to another organization. But I agree we probably shouldn't cascade at least not right now

File: h/migrations/versions/46a22db075d5_add_organization_id_to_group_table.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+revision = '46a22db075d5'
+down_revision = '628c53b07'
+
+
+def upgrade():
+    op.add_column('group', sa.Column(
+        'organization_id',
+        sa.Integer,
```
Comment (seanh) [inline h/migrations/versions/46a22db075d5_add_organization_id_to_group_table.py]: Looks like the import of `sa` is missing

**Pull request merged**